baseplate/datetime.py

```diff
@@ -0,0 +1,26 @@
+"""Extensions to the standard library `datetime` module."""
+
+from datetime import datetime
+from datetime import timezone
+
+
+def datetime_to_epoch_milliseconds(dt: datetime) -> int:
+    """Convert datetime object to epoch milliseconds."""
+    if not dt.tzinfo:
+        dt = dt.replace(tzinfo=timezone.utc)
+    return int(dt.timestamp() * 1000)
+
+
+def datetime_to_epoch_seconds(dt: datetime) -> int:
+    """Convert datetime object to epoch seconds."""
+    return int(datetime_to_epoch_milliseconds(dt) / 1000)
```

Comment:
You can use floor division `datetime_to_epoch_milliseconds(dt) // 1000` rather than casting to an `int` here